modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -121,219 +96,417 @@ Note that this applies to all variations of CSV files (see examples below for ot
 .Query
 [source, cypher]
 ----
-LOAD CSV FROM 'https://data.neo4j.com/bands/artists.csv' AS line
-CREATE (:Artist {name: line[1], year: toInteger(line[2])})
+LOAD CSV FROM 'https://data.neo4j.com/bands/artists.csv' AS row
+MERGE (:Artist {name: row[1], year: toInteger(row[2])})
```

Comment:
Here we could add a RETURN clause right? e.g 
MERGE (a:Artist {name: row[1], year: toInteger(row[2])})
RETURN a.name

Reply:
we should do the same for the previous query (local file) as well then. probably also a couple of others.